quickwit/quickwit-indexing/src/actors/doc_processor.rs

```diff
@@ -321,22 +388,22 @@ impl Handler<RawDocBatch> for DocProcessor {
             return Ok(());
         }
         let mut prepared_docs: Vec<PreparedDoc> = Vec::with_capacity(raw_doc_batch.docs.len());
-        for json_doc in raw_doc_batch.docs {
-            let json_doc_num_bytes = json_doc.len() as u64;
+        for doc in raw_doc_batch.docs {
+            let doc_num_bytes = doc.len() as u64;
 
-            match self.prepare_document(&json_doc, ctx) {
+            match self.prepare_document(doc, ctx) {
```

Comment:
nitpick: could be `process_document` that returns a `ProcessedDoc` to be consistend with `DocProccessorError`.

Reply:
Indeed. Will fix.